@RC@ is an unknown tag to libtool. Do not use it.

diff --git a/builds/unix/unix-cc.in b/builds/unix/unix-cc.in
index 5675866ea5..79e01d4a7d 100644
--- a/builds/unix/unix-cc.in
+++ b/builds/unix/unix-cc.in
@@ -92,7 +92,7 @@ CC    := $(LIBTOOL) --mode=compile $(CCraw)
 
 # Resource compiler to use on Cygwin/MinGW, usually windres.
 #
-RCraw := @RC@
+RCraw :=
 ifneq ($(RCraw),)
   RC := $(LIBTOOL) --tag=RC --mode=compile $(RCraw)
 endif
